### PR TITLE
add more llava && llava-next model support; enrich resutls json

### PR DESCRIPTION
**Pull request by KimBioInfoStudio**

### CODE REVIEW

File: examples/image-to-text/run_pipeline.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def main():
         args.prompt = "<image>\nUSER: What's the content of the image?\nASSISTANT:"
     elif args.prompt is None and model_type == "llava_next":
         args.prompt = "[INST] <image>\nWhat is shown in this image? [/INST]"
-        if args.model_name_or_path == "llava-hf/llava-v1.6-vicuna-13b-hf":
+        if args.model_name_or_path in ["llava-hf/llava-v1.6-vicuna-13b-hf", "llava-hf/llava-v1.6-vicuna-13b-hf"]:
```
Comment (yafshar) [inline examples/image-to-text/run_pipeline.py:112]: @KimBioInfoStudio the same name `llava-hf/llava-v1.6-vicuna-13b-hf` is repeated twice

Comment (KimBioInfoStudio) [reply examples/image-to-text/run_pipeline.py]: Thx, fixed

**Review by yafshar — APPROVED** [review]
LGTM!

**Comment by yafshar** [comment]
@regisss Could you please take a look at this?

**Comment by KimBioInfoStudio** [comment]
have already been done in #1120, so closed

**Pull request closed without merging**